### ISSUE TITLE
Gems management via CMD fails

### ISSUE THREAD

**Issue by LB-BartoszDudzinski** [o3de/o3de#6957]
**Describe the bug**
Managing gems via CMD fails with the following errors:

```
C:\o3de\o3de>scripts\o3de.bat disable-gem -pp C:\o3de\o3de\NewPro -gp C:\o3de\o3de\Gems\Multiplayer
[ERROR] o3de.disable_gem: Enabled gem file C:\o3de\o3de\**NewPro\Code**\enabled_gems.cmake is not present.

C:\o3de\o3de>scripts\o3de.bat enable-gem -pp C:\o3de\o3de\NewPro -gp C:\o3de\o3de\Gems\Multiplayer
Traceback (most recent call last):
  File "C:\o3de\o3de\scripts\o3de.py", line 95, in <module>
    ret = the_args.func(the_args) if hasattr(the_args, 'func') else 1
  File "C:\o3de\o3de\scripts\o3de\o3de\enable_gem.py", line 122, in _run_enable_gem_in_project
    args.enabled_gem_file)
  File "C:\o3de\o3de\scripts\o3de\o3de\enable_gem.py", line 97, in enable_gem_in_project
    project_enabled_gem_file.touch()
  File "C:\o3de\o3de\python\runtime\python-3.7.10-rev2-windows\python\lib\pathlib.py", line 1263, in touch
    fd = self._raw_open(flags, mode)
  File "C:\o3de\o3de\python\runtime\python-3.7.10-rev2-windows\python\lib\pathlib.py", line 1072, in _raw_open
    return self._accessor.open(self, flags, mode)
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\o3de\\o3de\\NewPro\\Code\\enabled_gems.cmake'
```
The enabled_gems.cmake file for a new project is located in the C:\O3DE-install\NewPro\Gem folder.

**Steps to reproduce**
1. Open the Command Prompt in the O3DE location.
2. Execute the command:
`scripts\o3de.bat enable-gem -pp <path to the custom project> -gp <path to the gem>`

**Expected behavior**
The gem is enabled in the custom project.

**Actual behavior**
An error related to the incorrect path is displayed.

**Screenshots/Video**

![gems](https://user-images.githubusercontent.com/86952009/149770810-0f715351-0b25-4f8b-8879-7746fd3680cc.JPG)

**Found in Branch**
Development (https://github.com/o3de/o3de/commit/76ff7aec29ad67f11f0dc84e47702cc8757dcdd2)

**Desktop/Device:**
- Device: PC
- OS: Windows
- Version 10
- CPU Intel Core i5-9600k
- GPU Nvidia GeForce RTX 2060
- Memory 16GB

**Comment by monroegm** [comment]
sig/core is responsible for o3de.bat

**Comment by LB-KacperBieniek** [comment]
Verified the issue is fixed in latest development (3bbb8d7) thanks to this pull request: https://github.com/o3de/o3de/pull/6976.
